### PR TITLE
Rs 2022 05 22

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Ciel/Day 2/02_01_CIEL02_1_1.txt [hunk 1]
```diff
@@ -36,22 +36,23 @@ I am sure that if I suggest this to the head of the family, she would accept it
 [sha:a2cb2b87b849cd5149e45e4caf8e112e7e654394]{
 -- Page 6, Offset 3631.
 -- 『そうですか。では兄さんの外食費はすべて、一月分の食券にしてお渡ししましょう。
-"I see. Then, I will give you a month's worth of meal coupons.
+'Is that so. Then please allow my to provide you with enough coupons to cover a full month's worth of eating out.
```
Comment (yk1512) [inline script/Ciel/Day 2/02_01_CIEL02_1_1.txt:39]: I believe 外食費 just means eating at school, which is out of the Tohno mansion. I wouldn't put 'eating out.' Feels like it contradicts the next line.

File: script/Ciel/Day 2/02_01_CIEL02_1_2.txt [hunk 2]
```diff
@@ -6,47 +6,52 @@ I'll just take it easy and buy some bread.
...
 -- Page 10002, Offset 3637.
 -- これからの財政事情は厳しいものとなる。
-My financial situation from now on is going to be tough.
+Money is going to be a lot tighter from now on.
 }
 [sha:709fd5bcaf02acc3827bf62955a367b48eda5b0b]{
 -- Page 10002, Offset 3638.
 -- 我が家のご当主さまは遊び心がないので、余分な収入など望むべくもない。悲しいが、そのあたりの希望はないのだ。
-The head of our family is not a playful person, so I shouldn't hope for any extra income. It's a sad reality, but one that's not going to change.
+Our family head is a harsh mistress, so I shouldn't hold out any hope for an extra allowance.
```
Comment (yk1512) [inline script/Ciel/Day 2/02_01_CIEL02_1_2.txt:14]: I think 'an' isn't needed here

File: script/Ciel/Day 2/02_01_CIEL02_1_2.txt [hunk 3]
```diff
@@ -236,17 +246,17 @@ I find myself with only two minutes left until the end of lunch break.
 [sha:fc84aad92bcce997481684fd7c1f3cf5c8f3cf91]{
 -- Page 28, Offset 3683.
 -- いいニュースも聞けたし、やかましい有彦はいないし、午後は平和に過ごせそうだ。
-I've heard some good news, and without that noisy Arihiko around, I think I can spend the afternoon in peace.
+I've just heard some good news, and without Arihiko around to cause trouble the rest of the afternoon should pass peacefully.
```
Comment (yk1512) [inline script/Ciel/Day 2/02_01_CIEL02_1_2.txt:249]: comma after trouble

File: script/Ciel/Day 2/02_01_CIEL02_1_2.txt [hunk 4]
```diff
@@ -331,17 +343,17 @@ The bell rings.
 [sha:939c5df2ababd7b15275f3dfb82e477629b77ff0]{
 -- Page 35, Offset 3702.
 -- 本気で嬉しいのだけど、素直に喜べないこの複雑さ。
-I'm overjoyed, but I can't accept these feelings so easily.
+I'm overjoyed, but this is actually makes things a little complicated.
```
Comment (yk1512) [inline script/Ciel/Day 2/02_01_CIEL02_1_2.txt:346]: remove 'is'

File: script/Ciel/Day 2/02_01_CIEL02_1_2.txt [hunk 5]
```diff
@@ -436,17 +456,17 @@ So all that talk about going out with her was off the mark huh?"
...
 -- Page 42, Offset 3724.
 -- 俺は無意識のうちにまっとうな価値観の人間……素で尊敬できてしまうような……を敬遠してしまうところがある。
-Subconsciously, I tend to avoid... people who have decent values... and whom I can respect.
+Subconsciously, I tend to avoid people who have decent values... people whom I can respect.
 }
 [sha:9e99bcd870e1b166511650f65805b666e27fd97c]{
 -- Page 42, Offset 3725.
 -- 嫌いなのではなく、どちらかというと怖くて近寄れないというか。
-It's not that I don't like them, it's more like I'm afraid to approach them.
+It's not that I don't like them, it's more like I'm afraid to get too close with them.
```
Comment (yk1512) [inline script/Ciel/Day 2/02_01_CIEL02_1_2.txt:469]: 'close to them' sounds more natural to me

File: script/Ciel/Day 2/02_01_CIEL02_1_3.txt [hunk 6]
```diff
@@ -146,27 +156,30 @@ I follow her example and sit down next to her at a slight distance.
...
 -- 「ふふふ、それはですね……」
-"Fufufu, well you see..."
+"Hehehe, well you see..."
 }
 [sha:a1057f2e5e10e2d6399b74f232029b5c62385e8e]{
 -- Page 60, Offset 3762.
 -- 先輩は嬉しさでとろけるように笑って紙袋を膝に置くと、中から四つの総菜パンを取りだした。
-Senpai smiles as if melting with joy, places the paper bag on her lap, and takes out four side dishes from inside.
+// 総菜パン doesn't have a great analog, but 'stuffed bread' used elsewhere in script
+Senpai is smiling fit to burst as she places the paper bag on her lap.
```
Comment (yk1512) [inline script/Ciel/Day 2/02_01_CIEL02_1_3.txt:181]: comma after smiling

File: script/Ciel/Day 2/02_01_CIEL02_1_3.txt [hunk 7]
```diff
@@ -246,17 +261,18 @@ For some reason, I specifically avoided asking 'Do you like curry?'
...
 -- 良かった……先輩も何が異常なのか察したようだ。
-Good... At least it looks like she's aware of it.
+Good... At least she seems to understand that it's slightly strange behavior.
 }
 [sha:a5e7c4afebd1449878671cf0cb6f206b3258e22d]{
 -- Page 69, Offset 3779.
 -- 「でも分かりますよ。好きなものってちょっと多いぐらい買い込んでおかないと不安になりますからね。
-"But I understand. If you don't buy enough of what you like, you get uneasy.
+// Is the idea of 不安 here that you buy a collection of bread for lunch and your date picks the one you were angling for?
+"Still, I get it. It's easy to feel nervous if you don't stock up on the things you enjoy.
```
Comment (yk1512) [inline script/Ciel/Day 2/02_01_CIEL02_1_3.txt:270]: I understand this feeling perfectly lol, it comes from a place of insecurity, the idea that you don't deserve good things. If a lot of good things happen to you, you get the feeling that you're going to receive karmic retribution and something bad will happen. In this case, buying stuff that you like invites that sense of incoming misfortune, is what they mean.

File: script/Ciel/Day 2/02_01_CIEL02_1_3.txt [hunk 8]
```diff
@@ -361,7 +386,7 @@ This guy saw me and Ciel-senpai together and snuck up on us...!
 [sha:e59e6e3f46c6c13e843bd47ccc939e34493262b7]{
 -- Page 80, Offset 3801.
 -- 「有彦……おまえ、なんでそう人の邪魔をすることばかりに長けているんだ？　前世からの呪いなのか？　それともそういう生き物なのか？」
-"Arihiko... why are you so good at getting in people's way? Is it a curse from a previous life? Or are you just that kind of creature?"
+"Arihiko... why are you so good at getting in people's way? Is it a curse from a previous life? Or is this just the way you are?"
```
Comment (yk1512) [inline script/Ciel/Day 2/02_01_CIEL02_1_3.txt:389]: I wonder if the 'curse' is actually Shiki saying that Arihiko cursed him specifically in a previous life.

File: script/Ciel/Day 2/02_01_CIEL02_1_3.txt [hunk 9]
```diff
@@ -371,17 +396,21 @@ This guy saw me and Ciel-senpai together and snuck up on us...!
...
-Besides, you're having lunch, so why don't you let me join you? I gave up my seat yesterday, so it's your turn to give up your seat this time, isn't it?"
+I mean c'mon, you're having lunch, so why don't you let me join you?
+ I gave up my seat yesterday, so now it's your turn to give up yours, don't you think?"
 }
 [sha:9e1c386fe75fd4c4d9f2348062799efdc3a1bbfb]{
 -- Page 82, Offset 3804.
 -- 有彦の中では昨日の昼食は“先輩のために席をキープしていた”コトになるらしい。まことに都合のいい頭である。
-In Arihiko's mind, yesterday's lunch was 'saving a seat for Senpai'. It is a very convenient way of thinking.
+Arihiko seems to have convinced himself that he was actually saving a seat for Senpai at lunch yesterday.
+ He has quiet the convenient memory.
```
Comment (yk1512) [inline script/Ciel/Day 2/02_01_CIEL02_1_3.txt:406]: quite

File: script/Ciel/Day 2/02_01_CIEL02_1_3.txt [hunk 10]
```diff
@@ -371,17 +396,21 @@ This guy saw me and Ciel-senpai together and snuck up on us...!
...
 }
 [sha:4554ac57a0eeb2b0ed4d7b615c49465a640a6d42]{
 -- Page 83, Offset 3805.
 -- 「……はあ。すみません先輩。うちのクラスのバカが失礼を働きました。こんなコト言ってますけど、昨日はいちおう役に立ったんで同席させて……先輩？」
-"...%{i}Sigh%{/i}. Sorry, Senpai. This idiot from my class is being rude. I know I'm asking a lot, but he was useful yesterday, so could you please let him si― Senpai?"
+"...%{i}Sigh%{/i}. Sorry, Senpai.
+ This idiot from my class is being rude.
+ I know I'm asking a lot, but did come in handy yesterday, so could you please let him si― Senpai?"
```
Comment (yk1512) [inline script/Ciel/Day 2/02_01_CIEL02_1_3.txt:413]: **he** did come in handy

File: script/Ciel/Day 2/02_01_CIEL02_1_3.txt [hunk 11]
```diff
@@ -421,17 +450,23 @@ Senpai's left jab cuts through the air with a whoosh.
...
 -- Page 88, Offset 3815. Mods: @k, @e.
 -- 「お、おう、なんだこれ冷や汗止まらねえよヤバイマジヤバイ拳圧で先輩が大きく見える、
-"O-Oh, what the hell is this? I can't stop sweating. This is bad, real bad. The pressure is making her fists look bigger.
+// 拳圧 seems to maybe be the anime fist shockwave?
+// https://spirits.appirits.com/doruby/10041/
+// https://detail.chiebukuro.yahoo.co.jp/qa/question_detail/q1139422521
+"O-Oh, what the hell is this?
+ I can't stop sweating.
+ This is bad, real bad.
+ The shock waves from of her fists are making her look bigger.
```
Comment (yk1512) [inline script/Ciel/Day 2/02_01_CIEL02_1_3.txt:469]: remove of

File: script/Ciel/Day 2/02_01_CIEL02_1_1.txt [hunk 12]
```diff
@@ -26,7 +26,7 @@ After all, as of yesterday, I am the eldest son of the Tohno family.
 [sha:c858178efe048ed3a818fd333577ff56c703e6f9]{
 -- Page 5, Offset 3629.
 -- 『学食は毎日、いちばんリッチなものにしたい』
-"I want to have the most expensive meal in the cafeteria every day."
+'I want to have the most expensive meal in the cafeteria every day.'
```
Comment (Volvo-hackangel) [inline script/Ciel/Day 2/02_01_CIEL02_1_1.txt:29]: why the change? it's a voiced line

File: script/Ciel/Day 2/02_01_CIEL02_1_3.txt [hunk 13]
```diff
@@ -861,17 +910,18 @@ Come to think of it, I've never spent time in the courtyard like this before, co
 [sha:3e6b08507956874a781a68ebf066b1244923a81f]{
 -- Page 119, Offset 3901.
 -- あんまり中庭に出た事がなかったから。ああでも、前にもこんなコトがあったような。シエル先輩と会った時でしたっけ？」
+// Wait, is this a reference to the OG?
```
Comment (yk1512) [inline script/Ciel/Day 2/02_01_CIEL02_1_3.txt:913]: :aokothink:

File: script/Ciel/Day 2/02_01_CIEL02_1_4.txt [hunk 14]
```diff
@@ -76,32 +77,33 @@ Because the staff room is so homey..."
 [sha:d7b7fa15a1db4a2a729f1ac5515ec6fd0b14b65a]{
 -- Page 126, Offset 3922.
 -- とか言われたら目が点になるでしょ？
-It's a little bit of a shock, isn't it?
+You can't help but be taken aback when someone says stuff that to you, right?
```
Comment (yk1512) [inline script/Ciel/Day 2/02_01_CIEL02_1_4.txt:80]: stuff like that?

File: script/Ciel/Day 2/02_01_CIEL02_1_4.txt [hunk 15]
```diff
@@ -116,32 +118,36 @@ A teacher must be tough, positive, and smiling.
...
 -- 私も笑顔には自信があるけど、先生方にはちょっと負けるなー」
-I'm confident in my smile too, but I'm not as good as the other teachers."
+I'm pretty confident in my smile,
+ but I'm still feeling a little put out after seeing the other teachers."
 }
 [sha:e3e7bbe25ab2150dcd1f302cc4bf866a6a54f7b5]{
 -- Page 129, Offset 3931.
 -- そう言いながら、ノエル先生は手にしていた携帯端末をスーツのポケットに仕舞った。
-As she says this, Noel-sensei puts the mobile phone she was holding in her hand into her suit pocket.
+As she says this, Noel-sensei slips the mobile phone she was holding hand into her suit pocket.
```
Comment (yk1512) [inline script/Ciel/Day 2/02_01_CIEL02_1_4.txt:127]: remove hand

File: script/Ciel/Day 2/02_01_CIEL02_1_4.txt [hunk 16]
```diff
@@ -16,7 +17,7 @@ Whether I want to go to the cafeteria or the store, I have to get down to the fi
 [sha:a09136e79fda54d795a65f94a0c39c3d93467cc9]{
 -- Page 122, Offset 3910.
 -- 一階に下りる際、不思議なものと遭遇した。
-On my way down to the first floor, I encounter something strange.
+When I go to head down to the stairs, I come across something strange.
```
Comment (Volvo-hackangel) [inline script/Ciel/Day 2/02_01_CIEL02_1_4.txt:20]: down the stairs

File: script/Ciel/Day 2/02_01_CIEL02_1_4.txt [hunk 17]
```diff
@@ -26,17 +27,17 @@ The new teacher, Noel-sensei, is standing in the hallway doing nothing.
...
 -- Page 123, Offset 3912.
 -- 昼休み、教員が廊下にいるのは珍しい。
-It is unusual for a teacher to be in the hallway during lunchtime.
+It's unusual to find teachers in the hallway during lunchtime.
 }
 [sha:8f9d221524a2e2bbfaf0495d5c5f8667138e39e0]{
 -- Page 123, Offset 3913.
 -- 我が校では、教員は職員室か準備教室か、食堂の教員スペースで昼食を摂るのが常だ。
-At our school, teachers usually have lunch in the staff room, the preparation room, or the teacher's area in the cafeteria.
+The teachers at our school usually have lunch in the staff room, prep room or the designated staff area in the cafeteria.
```
Comment (Volvo-hackangel) [inline script/Ciel/Day 2/02_01_CIEL02_1_4.txt:35]: maybe keep the comma?
prep room,

File: script/Ciel/Day 2/02_01_CIEL02_1_3.txt (hunk 6 above)
Comment (rschlaikjer) [reply script/Ciel/Day 2/02_01_CIEL02_1_3.txt:181]: (smiling fit to burst) is a phrase, not sure I'd break it up with a comma

Comment (yk1512) [reply script/Ciel/Day 2/02_01_CIEL02_1_3.txt:181]: alright

File: script/Ciel/Day 2/02_01_CIEL02_1_3.txt (hunk 7 above)
Comment (yk1512) [reply script/Ciel/Day 2/02_01_CIEL02_1_3.txt:270]: actually, you are right :aichaneheh:

File: script/Ciel/Day 2/02_01_CIEL02_1_2.txt [hunk 18]
```diff
@@ -416,7 +436,8 @@ So, what do you think? Are you going to go out with her? Since she's made this m
 [sha:0f7e73b71e3c89f38d5a79a04d45be59c089f904]{
 -- Page 40, Offset 3719.
 -- 「いや、それが……なんて言うか、弓塚さんを見てるとなんでか悲しくなるんだ、俺」
-"No, that's... how do I say this, looking at her makes me sad for some reason."
+"Well, that's because... how do I say this,
```
Comment (Avuxo) [inline script/Ciel/Day 2/02_01_CIEL02_1_2.txt:439]: how do i put this

File: script/Ciel/Day 2/02_01_CIEL02_1_2.txt [hunk 19]
```diff
@@ -436,17 +457,17 @@ So all that talk about going out with her was off the mark huh?"
...
 -- Page 41, Offset 3723.
 -- しかしこればかりはどうしようもない。
-But there is nothing I can do about it.
+But it's not as though I can help it.
 }
 [sha:f23af849203809ceb1e99906ab382adc5c7ad40d]{
 -- Page 42, Offset 3724.
 -- 俺は無意識のうちにまっとうな価値観の人間……素で尊敬できてしまうような……を敬遠してしまうところがある。
-Subconsciously, I tend to avoid... people who have decent values... and whom I can respect.
+Subconsciously, I tend to avoid people who have decent values... people whom I can respect.
```
Comment (Avuxo) [inline script/Ciel/Day 2/02_01_CIEL02_1_2.txt:465]: should be "who" now

File: script/Ciel/Day 2/02_01_CIEL02_1_3.txt [hunk 20]
```diff
@@ -11,57 +11,65 @@ As soon as I step into the hallway, I run into a familiar upperclassman.
...
+"Senpai, you're an upperclassman, you can't just come waltzing into the second-year classrooms.
+ If you're going to invite someone to join you, why not one of your third-year classmates?
+ I'm sure they'd make for a livelier conversation partner."
 }
 [sha:9897c5d50f4fe98195127bdb7ca831d977133288]{
 -- Page 52, Offset 3740.
 -- 「そうなんですけど、今日は年下の男の子とお話がしたい気分なんです。なんとなくなので、理由は聞かないでください」
-"That's true, but I'm in the mood to talk to a younger boy today. I just feel like it, so please don't ask me why."
+"That may be true, but I'm in the mood to talk to a younger boy today.
+ It's just a whim of mine, so please don't ask me to explain my reasons."
```
Comment (Avuxo) [inline script/Ciel/Day 2/02_01_CIEL02_1_3.txt:67]: i'd prefer "I just felt like it"

File: script/Ciel/Day 2/02_01_CIEL02_1_3.txt (hunk 6 above)
Comment (Avuxo) [reply script/Ciel/Day 2/02_01_CIEL02_1_3.txt:181]: not an american phrase. In fact, I've never heard it before as far as i can remember
https://www.ldoceonline.com/dictionary/fit-to-burst

File: script/Ciel/Day 2/02_01_CIEL02_1_3.txt [hunk 21]
```diff
@@ -706,7 +749,10 @@ Ciel-senpai, please don't fall for Arihiko's nonsense!"
 [sha:be405465901e8da93d53b2faa5968fb70312b8f7]{
 -- Page 108, Offset 3870.
 -- 「ですよねー。遠野くんがあんまりにも無防備にひょこひょこ歩いていたので、わたしもつい誘われてしまったのです。なんというか、雪原で兎にあったような気分でした」
-"That's right. Tohno-kun was walking so unguardedly, I was tempted to take him with me. It was like running into a rabbit in a snowfield."
+// Damn he fucking loves this 雪原.*兎 imagery, didn't realize it started so early
```
Comment (Avuxo) [inline script/Ciel/Day 2/02_01_CIEL02_1_3.txt:752]: one time in highschool i accidentally used the same analogy 3 times in an essay

File: script/Ciel/Day 2/02_01_CIEL02_1_3.txt [hunk 22]
```diff
@@ -726,17 +772,18 @@ I want to object, but I'm afraid I'll end up bringing unnecessary trouble upon m
 [sha:f5b059f7733c8bfeb008e8755925f2f477bbc2ce]{
 -- Page 109, Offset 3874.
 -- そして着々と減っていくカレーパン。
-And so, the number of curry bread dwindles.
+And so, the number of curry bread steadily dwindles.
```
Comment (Avuxo) [inline script/Ciel/Day 2/02_01_CIEL02_1_3.txt:775]: "number of bread" is kinda awkward. maybe "number of curry buns"

File: script/Ciel/Day 2/02_01_CIEL02_1_3.txt (hunk 13 above)
Comment (Avuxo) [reply script/Ciel/Day 2/02_01_CIEL02_1_3.txt:915]: I've noticed a couple of lines like this and it scares me. red garden is gonna ff7r this shit i can feel it. higurashi did it too

**Review by Avuxo — APPROVED** [review]
what a pr

File: script/Ciel/Day 2/02_01_CIEL02_1_3.txt (hunk 6 above)
Comment (rschlaikjer) [reply script/Ciel/Day 2/02_01_CIEL02_1_3.txt:181]: :soubrit: 

Positively beaming? Or is that too UK as well?

File: script/Ciel/Day 1/01_01_CIEL01_3_3_2.txt [hunk 23]
```diff
@@ -270,37 +282,45 @@ The seriousness of her voice makes me straighten up.
...
 -- Page 254, Offset 1095.
 -- 本当の両親ではない父親と母親。
-A mother and father who were not my real parents.
+They may have been my mother and father, but they weren't my real parents.
 }
 [sha:86ebfd9f6f22b1c3970de77a5d36de06cde801d2]{
 -- Page 254, Offset 1096.
 -- 親戚の家であるはずの見知らぬ建物。
-A strange building that was supposed to be a relative's house.
+// Not 100% sure what であるはず is adding here
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 1/01_01_CIEL01_3_3_2.txt:304]: Yeah, it just seems like a weird way to put it since like... he knows it's the Arimas' house and that they're relatives? I guess do you think it's worth trying to encode?

File: script/Ciel/Day 2/02_01_CIEL02_1_3.txt (hunk 6 above)
Comment (Avuxo) [reply script/Ciel/Day 2/02_01_CIEL02_1_3.txt:181]: positively beaming is fine

File: script/Ciel/Day 1/01_01_CIEL01_3_3_2.txt (hunk 23 above)
[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Ciel/Day 1/01_01_CIEL01_3_3_2.txt:304]: not really, no. I think this line is fine as-is
